### ISSUE TITLE
openscad-dev: checkver needs updated

### ISSUE THREAD

**Issue by amreus** [ScoopInstaller/Versions#385]
https://github.com/ScoopInstaller/Versions/blob/46bdc5a9332554ab64b3e62a2b137acdfcbe60fa/bucket/openscad-dev.json#L23

It looks like the upstream url has changed.

Would it be better to just use the actual file list instead? https://files.openscad.org/snapshots/

as well as updating `checkver.regex` as appropriate.

**Comment by rashil2000** [comment]
> Would it be better to just use the actual file list instead? https://files.openscad.org/snapshots/
> 
> as well as updating `checkver.regex` as appropriate.

Yes! Please make a PR!

**Comment by amreus** [comment]
Yep, just logging for when I get back to it

**Comment by amreus** [comment]
It turns out there might be a good a reason the author of `openscad-dev.json` is parsing the "hidden" `.js` file to get the latest version.

The version numbers for the 32bit and 64bit .zip archives are different. 

I'm not sure how to write the regex to capture the versions.

**Comment by rashil2000** [comment]
The chekver will be this:
```json
"checkver": {
    "url": "https://files.openscad.org/snapshots",
    "regex": "(?sm)OpenSCAD-(?<version32>[\\w.]+)-x86-32\\.zip\\.sha512.*?OpenSCAD-(?<version64>[\\w.]+)-x86-64-Installer\\.exe",
    "reverse": true
}
```

But this is not very reliable, as the order of 32bit entries and 64bit entries for the same day is not consistent. A sure shot way would be to use the 2 different links for the architectures - https://files.openscad.org/snapshots/.snapshot_win32.js and https://files.openscad.org/snapshots/.snapshot_win64.js - in a manner similar to vlc-nightly:

https://github.com/ScoopInstaller/Versions/blob/608574d640c632172dfb042d49e0862a577d4f1e/bucket/vlc-nightly.json#L33-L50

**Comment by amreus** [comment]
This could also be resolved by ScoopInstaller/Scoop#4611

**Comment by rashil2000** [comment]
> This could also be resolved by [ScoopInstaller/Scoop#4611](https://github.com/ScoopInstaller/Scoop/issues/4611)

Yes, but I don't know when that will be fixed

**Comment by amreus** [comment]
> Yes, but I don't know when that will be fixed
